There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs

```diff
@@ -1,8 +1,6 @@
 // # A program that resolves the "There is no spoon - Episode 1" problem from [Coding game website](https://www.codingame.com/home)
```

Comment:
I would use "solves", not "resolves".

Reply:
The two tend to be used interchangeably. I have changed it.